telegram/ext/_applicationbuilder.py

```diff
@@ -478,39 +546,104 @@ def get_updates_request(self: BuilderType, get_updates_request: BaseRequest) ->
     def get_updates_connection_pool_size(
         self: BuilderType, get_updates_connection_pool_size: int
     ) -> BuilderType:
+        """Sets the size of the connection pool to be used for the
+        :paramref:`~telegram.request.HTTPXRequest.connection_pool_size` parameter of
+        :attr:`telegram.Bot.request` for :meth:`telegram.Bot.get_updates`. Defaults to ``1``.
```

Comment:
`Bot.request` is the BaseRequest instance for non-get_updates requests … Currently the get_updates instance is not exposed (IISC `Bot.request` is currently only really there because `File.download` accesses it).

same below